ge/org.osate.ge.errormodel/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -4,4 +4,4 @@ org.eclipse.jdt.core.compiler.codegen.targetPlatform=1.8
 org.eclipse.jdt.core.compiler.compliance=1.8
 org.eclipse.jdt.core.compiler.problem.assertIdentifier=error
 org.eclipse.jdt.core.compiler.problem.enumIdentifier=error
-org.eclipse.jdt.core.compiler.source=1.8
```

Comment:
I cannot tell what changed here, is this correct?

Reply:
Yes